core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/dot-edit-content-category-field.component.ts

```diff
@@ -69,6 +65,12 @@ export class DotEditContentCategoryFieldComponent implements OnInit {
      * Disable the button to open the sidebar
      */
     disableSelectCategoriesButton = signal(false);
+    /**
+     * Represents the variable '$maxChips' which is of type 'signal<number>'.
+     *
+     * @memberof DotEditContentCategoryFieldComponent
+     */
+    $maxChips = signal(MAX_CHIPS);
```

Comment:
As mentioned above, this could be direct a default value in the other component (the input)